samples/snippets/src/main/java/com/example/contactcenterinsights/CreatePhraseMatcherAllOf.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.contactcenterinsights;
+
+// [START contactcenterinsights_create_phrase_matcher_all_of]
+
+import com.google.cloud.contactcenterinsights.v1.ContactCenterInsightsClient;
+import com.google.cloud.contactcenterinsights.v1.ExactMatchConfig;
+import com.google.cloud.contactcenterinsights.v1.LocationName;
+import com.google.cloud.contactcenterinsights.v1.PhraseMatchRule;
+import com.google.cloud.contactcenterinsights.v1.PhraseMatchRuleConfig;
+import com.google.cloud.contactcenterinsights.v1.PhraseMatchRuleGroup;
+import com.google.cloud.contactcenterinsights.v1.PhraseMatcher;
+import java.io.IOException;
+
+public class CreatePhraseMatcherAllOf {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace this variable before running the sample.
+    String projectId = "my_project_id";
+
+    createPhraseMatcherAllOf(projectId);
+  }
+
+  public static PhraseMatcher createPhraseMatcherAllOf(String projectId) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (ContactCenterInsightsClient client = ContactCenterInsightsClient.create()) {
+      // Construct a phrase matcher that matches all of its rule groups.
+      PhraseMatcher.Builder phraseMatcher =
+          PhraseMatcher.newBuilder()
+              .setDisplayName("NON_SHIPPING_PHONE_SERVICE")
+              .setTypeValue(1)
```

Comment:
Can we use a human-readable value here? Same comment on the other usages.

Reply:
we have to use an integer here because we're setting `type` which is an enum, and the Java-generated enum code only supports a `set` method that takes an integer https://developers.google.com/protocol-buffers/docs/reference/java-generated#enum-fields

Reply:
Bummer, thanks!